fastapi_asyncapi/schema.py

```diff
@@ -233,8 +245,28 @@ class OAUHTFlows(BaseModel):
     authorizationCode: Optional[OAUHTFlow]
 
 
+class SecuritySchemesType(Enum):
```

Comment:
https://fastapi-utils.davidmontague.xyz/user-guide/basics/enums/

```suggestion
class SecuritySchemesType(str, Enum):
```